### PR TITLE
refactor: replace `pkg/errors` with stdlib `errors`

### PR DESCRIPTION
**Pull request by Juneezee**
Since Go 1.13, the `errors` and `fmt` standard library supports unwrapping and wrapping errors. See https://go.dev/blog/go1.13-errors and https://pkg.go.dev/errors

The `github.com/pkg/errors` package has now been archived and no longer in active development. This commit replaces all `errors.Wrap` and `errors.Wrapf` with `fmt.Errorf` and removes the dependency on `github.com/pkg/errors`.

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: dockertest.go [hunk 1]
```diff
@@ -579,7 +579,7 @@ func (d *Pool) CurrentContainer() (*Resource, error) {
 	case *dc.NoSuchContainer:
 		return nil, ErrNotInContainer
 	default:
-		return nil, errors.Wrap(err, "")
+		return nil, fmt.Errorf("%w", err)
```
Comment (obalunenko) [inline dockertest.go]: Could be just err returned, this wrapping is not needed

Comment (Juneezee) [reply dockertest.go]: Updated. Please take a look again. Thank you.

**Review by obalunenko — APPROVED** [review]
LGTM

**Review by aeneasr — CHANGES_REQUESTED** [review]
Thank you! Unfortunately Go‘s standard package still does not have stack traces. Replacing it with Go stderr will remove the stack traces which we need for debugging - thus I can’t accept this PR, sorry :(

**Pull request closed without merging**